build.gradle

```diff
@@ -117,6 +117,7 @@ ext.deps = [
     slf4j                : 'org.slf4j:slf4j-api:' + versions.slf4j,
     slf4jLog4j           : 'org.slf4j:slf4j-log4j12:' + versions.slf4j,
     sparkCore            : 'org.apache.spark:spark-core_2.10:1.4.0',
+    springTest           : 'org.springframework:spring-test:3.2.18.RELEASE',
```

Comment:
Why do we need to add this dependency?

Reply:
For mock implementations of servlet request & response. Those are big interfaces and tedious to mock with mockito.